mito-ai/src/utils/userSignupState.ts

```diff
@@ -39,9 +45,10 @@ export const checkUserSignupState = async (): Promise<UserSignupState> => {
 
         if (!hasThreads) {
             return {
-                isSignedUp: false,
+                isSignedUp: hasSoftSignup,
                 hasEmail: false,
-                hasChatHistory: false
+                hasChatHistory: false,
+                hasSoftSignup: hasSoftSignup
```

Comment:
It doesn't look like we are ever using these different fields. Should we just simplify this to: 

1. Calling the function `isUserSignedUp`
2. It just returns True or False